src/main/java/org/jenkinsci/plugins/docker/traceability/core/DockerTraceabilityRootAction.java

```
@@ -568,18 +571,62 @@ private synchronized void load() {
             LOGGER.log(Level.SEVERE, "Failed to load the configuration from config path = "+config,e);
         }
     }
-    
+
+    /**
+     * Represents the different response formats (JSON, pretty JSON, XML).
+     */
+    private enum ResponseFormat {
+        JSON,
+        PRETTYJSON,
+        XML;
+
+        private static final ResponseFormat DEFAULT = JSON;
+
+        public static ResponseFormat fromString (final String format) {
+            if (format == null) {
+                return DEFAULT;
+            }
+            if (format.equals("json")) {
+                return JSON;
+            } else if (format.equals("json-pretty")) {
+                return PRETTYJSON;
+            } else if (format.equals("xml")) {
+                 // Related to https://issues.jenkins-ci.org/browse/JENKINS-28727
+                return DEFAULT;
+            } else {
+                return DEFAULT;
```

I would throw an error in the case of the unsupported format